exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
+ */
+public class SimpleLinkedList {
+
+    class Element {
+
+        int value;
+        Element next;
+
+        Element(int value) {
+            this.value = value;
+        }
+    }
+
+    private Element head;
+    private int size = 0;
+
+    public SimpleLinkedList() {
```

Comment:
Java will provide a [default constructor](https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.8.9), so this isn't necessary.